deployments/helm/onos-gui/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "onos-gui.fullname" . }}-ingress
```

Comment:
for consistency you might want to add the namespace also here

Reply:
done